redis/Redis.php

```diff
@@ -2132,7 +2132,7 @@ public function sort( $key, $option = null ) {}
      * - vm_enabled
      * - role
      * @link    https://redis.io/commands/info
-     * @return string
+     * @return string|array
```

Comment:
```suggestion
     * @return array
```
According to the [docs](https://github.com/phpredis/phpredis#info) and to the [sources](https://github.com/phpredis/phpredis/blob/d7450b2f59700e4662624317438c456a0dd36165/library.c#L853) this method can return only `array`, not `string`.